.github/workflows/ci-nix.yml

```diff
@@ -19,6 +19,16 @@ on:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
+  self-care:
+    name: Flake self-check
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Check Nix flake inputs
+        uses: DeterminateSystems/flake-checker-action@v5
+        with:
+          fail-mode: true
```

Comment:
Will this block merging?

Reply:
Nope

Reply:
Only the workflows explicitly listed in github settings for the repo are required.